cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/token/TokenRangeMapping.java

```diff
@@ -113,12 +139,10 @@ Multimap<I, Range<BigInteger>> tokenRangesByInstance(List<ReplicaInfo> writeRepl
     }
 
     public TokenRangeMapping(Partitioner partitioner,
-                             ReplicationFactor replicationFactor,
```

Comment:
Since `TokenRangeMapping` now can be consolidated, it no longer makes sense to contain ReplicationFactor, which is tie to cluster. 
The `TokenRangeMapping` or topology is only to describe the ring layout. `ReplicationFactor` is for consistency coordination. It is not part of `TokenRangeMapping` at the first place.